sbnana/SBNAna/anademo/CRUMBS_demo/CRUMBS_helper.h

```diff
@@ -0,0 +1,70 @@
+const Cut kNonUnambiguousSlice([](const caf::SRSliceProxy *slc)
```

Comment:
I would prefer to add this file into the compiled code (as a .cxx and .h file) under something like SBAna/Cuts/CrumbsCuts.h
That makes it easier to find, and also prevents it bit-rotting (because when it does it will break the build)

Reply:
Could do, I just wanted to make it clear that these are demo cuts and I would rather people chose the right cut value for their analysis themselves, not necessarily the values I've used here.

Reply:
put Demo in the filename and/or cut names?